README.md

```diff
@@ -25,9 +25,16 @@ TiPocket is inspired by [jepsen-io/jepsen](https://github.com/jepsen-io/jepsen),
 * ~~delay_tikv, delay_pd, errno_tikv, errno_pd, mixed_tikv, mixed_pd: Inject IO-related fault.~~
 * small_skews, subcritical_skews, critical_skews, big_skews, huge_skews: Clock skew, small_skews ~100ms, subcritical_skews ~200ms, critical_skews ~250ms, big_skews ~500ms and huge_skews ~5s.
 
-## Place your test case
+## Create a new case
 
-The new test case should be placed on ./testcase directory and create a nested sub-module avoiding go module conflict mutually.
+run `make init c=$case`, for example:
+
+```sh
+$ make init c=demo
+GO15VENDOREXPERIMENT="1" CGO_ENABLED=1 GOOS= GOARCH=amd64 GO111MODULE=on go build -ldflags '-s -w -X "github.com/pingcap/tipocket/pkg/test-infra/fixture.BuildTS=2021-02-05 07:13:54" -X "github.com/pingcap/tipocket/pkg/test-infra/fixture.BuildHash=a70411f45605864da28a5000aff72a226a1ab27f"'  -o bin/tipocket cmd/tipocket/*.go
+bin/tipocket init -c demo
+create a new case `demo`: testcase/demo
+```
```

Comment:
The test case generated has the following problem
```
~/c/t/t/bigregion tipocket_scaffold* > bin/bigregion -tidb-server 127.0.0.1:4000
2021/02/19 00:30:08 fixture.go:241: [info] Git Commit Hash: 56f06f87bae08ab648908e81b3d5ece3ce2882eb
2021/02/19 00:30:08 fixture.go:242: [info] UTC Build Time: 2021-02-19 12:28:26
W0219 00:30:08.112673   21678 client_config.go:543] Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work.
W0219 00:30:08.112722   21678 client_config.go:548] error creating inClusterConfig, falling back to default config: unable to load in-cluster configuration, KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT must be defined
2021/02/19 00:30:08 client.go:109: [warning] build config failed: invalid configuration: no configuration has been provided
2021/02/19 00:30:08 client.go:47: [warning] error creating kube-client: must provide non-nil rest.Config to client.New
2021/02/19 00:30:08 suit.go:87: [info] deploy cluster success, node:[node[comp=tidb,ip=127.0.0.1:4000]], client node:[client_node[comp=tidb,ip=127.0.0.1:4000]]
2021/02/19 00:30:08 control.go:88: [fatal] failed to create directory /var/run/tipocket-logs error is mkdir /var/run/tipocket-logs: permission denied
```

Reply:
resolved on 5ac0fdded38abe17c3ae27f9c7fabac2226932b3

Reply:
Now example run result:
```log
2021/02/19 11:49:15 control.go:98: [info] start controller with &{Mode:Standard or default mode DB: Nodes:[node[comp=tidb,ip=127.0.0.1:4000]] ClientNodes:[client_node[comp=tidb,ip=127.0.0.1:4000]] ClientCount:1 ChaosNamespace: RunRound:1 RunTime:1h40m0s RequestCount:0 History: ClientConfig:<nil>}
2021/02/19 11:49:15 control.go:347: [info] begin to set up database
2021/02/19 11:49:15 control.go:349: [info] begin to set up database on node[comp=tidb,ip=127.0.0.1:4000]
2021/02/19 11:49:15 control.go:370: [info] begin to set up client
2021/02/19 11:49:15 control.go:373: [info] begin to set up db client for node client_node[comp=tidb,ip=127.0.0.1:4000]
2021/02/19 11:49:15 client.go:42: [info] start to setup client...
2021/02/19 11:49:15 panic_check.go:34: [warning] plugin panic check won't work
2021/02/19 11:49:15 control.go:297: [info] run client 0...
2021/02/19 11:49:15 client.go:80: [info] everything is ok!
2021/02/19 11:49:15 control.go:382: [info] begin to tear down client
2021/02/19 11:49:15 control.go:385: [info] begin to tear down db client for node client_node[comp=tidb,ip=127.0.0.1:4000]
2021/02/19 11:49:15 control.go:359: [info] begin to tear down database
2021/02/19 11:49:15 control.go:361: [info] being to tear down database on node[comp=tidb,ip=127.0.0.1:4000]
2021/02/19 11:49:15 suit.go:151: [info] tear down cluster...
```